be/src/exec/table_function_node.cpp

```diff
@@ -105,115 +101,7 @@ Status TableFunctionNode::open(RuntimeState* state) {
 }
 
 Status TableFunctionNode::get_next(RuntimeState* state, ChunkPtr* chunk, bool* eos) {
-    RETURN_IF_CANCELLED(state);
```

Comment:
Please don't disable this feature in this feature. You can disable it in a separated PR.

Reply:
Restored, PTAL